packages/frontend/src/hooks/useQuestionContent.ts

```diff
@@ -0,0 +1,40 @@
+import { CoreFetcher } from "@carrot-kpi/sdk";
+import { useEffect, useState } from "react";
+
+export function useQuestionContent(cid: string | undefined): {
```

Comment:
I'm thinking this could be a more generic hook in the React library, no? Something like `usDecentralizedStorageContent`? I wouldn't call it `useIpfsContent` for now because I think in the future we could support Arweave or some other decentralized storage solutions.